modules/util/tzdata/execute.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+for candidate in yum dnf microdnf; do
+    if command -v "$candidate"; then
+        mgr="$(command -v "$candidate")"
+        "$mgr" reinstall tzdata -y && \
```

Comment:
Should this re-install `tzdata` and `tzdata-java`?

Reply:
`tzdata-java` appears to be intact (`rpm -V` is quiet). Unless there's something crucial in one of its scriptlets relying on `tzdata` (I'll take a look), there's no need.

Reply:
OK. Thanks for checking.

Reply:
no scriptlets.